### PR TITLE
Dbot- Quick Actions and EvenOdd Run 

### PR DESCRIPTION
**Pull request by thivya-deriv**
- Added a test to check on the even and odd run on dbot
- Added a test to check quick strategy and bot builder redirection from dashboard 
Card : https://app.clickup.com/t/20696747/BOT-1313

### CODE REVIEW

File: cypress/e2e/wip/dbot/regression-tests/DashboardQuickAction.cy.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+describe("QATEST-4128: Dashboard quick action to Quick Strategy and Bot Builder", () => {
+  const common = new Common();
+  const botDashboard = new BotDashboard();
+  const botBuilder = new BotBuilder();
+  
+  beforeEach(() => {
+    cy.c_login();
+    cy.c_visitResponsive("/bot", "large");
+    common.skipTour();
+    cy.close_notification_banner();
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/dbot/regression-tests/DashboardQuickAction.cy.js:16]: Add a TODO here; Deborah is modifying the name in her PR and we will need to update it later

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]
Please remove the semi-colons since we won't be using them in our framework 🙏

**Review by abuzar-deriv — APPROVED** [review]
Looks good.

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**